backend/manager/modules/restapi/jaxrs/src/main/resources/v3/capabilities.xml

```diff
@@ -575,7 +575,6 @@
             <custom_property name="sndbuf" regexp="^[0-9]+$"/>
             <custom_property name="vhost" regexp="^(([a-zA-Z0-9_]*):(true|false))(,(([a-zA-Z0-9_]*):(true|false)))*$"/>
             <custom_property name="viodiskcache" regexp="^(none|writeback|writethrough)$"/>
-            <custom_property name="mdev_type" regexp="^.*$"/>
```

Comment:
@mwperina @oliel can we safely drop it without breaking backward compatibility?

Reply:
@mz-pdm would a request that comes from a client that has not been upgraded and still specifies mdev_type fail? if so, that's another flow in which we need to convert the mdev_type to device(s) like reading an OVF

Reply:
The PR implements the conversion in OvfReader.  Reading snapshots and imports from OVA work for me in a 4.6 cluster.

Reply:
As was pointed out by @ahadas elsewhere, we need to support old REST API clients.  That means not removing this regexp and handling this property when present (similarly to what we do in OvfReader).

Reply:
I added a preparation commit for this. However, it doesn't work yet completely -- the device changes are not applied. This can be resolved in a followup PR; this one still improves master and doesn't make it worse than it currently is.

Reply:
I believe this has been fixed in https://github.com/oVirt/ovirt-engine/pull/190/commits/99133bd33f4773ad9c9485757153800672578e14 (thanks @ahadas!).